native-vm-just-run-ninja/build.ninja

```diff
@@ -0,0 +1,19 @@
+cxx = g++
```

Comment:
Can you give me an overview of how ninja is integrated into the build or test pipeline?

Reply:
In some tests, whenever cmake is invoked with `-G Ninja` to force ninja as a generator for performance reasons.

Reply:
Actually i needed to change the invocation of a few things and did that on the side.